data_pipeline/kubernetes/kubernetes_pipeline_config.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass
+from typing import Sequence
+
+from data_pipeline.kubernetes.kubernetes_pipeline_config_typing import (
+    KubernetesPipelineConfigDict,
+    MultiKubernetesPipelineConfigDict
+)
+
+
+@dataclass(frozen=True)
+class KubernetesPipelineConfig:
+    data_pipeline_id: str
+    image: str
+    arguments: str
+
+    @staticmethod
+    def from_dict(
+        pipeline_config_dict: KubernetesPipelineConfigDict
+    ) -> 'KubernetesPipelineConfig':
+        return KubernetesPipelineConfig(
+            data_pipeline_id=pipeline_config_dict['dataPipelineId'],
+            image=pipeline_config_dict['image'],
+            arguments=' '.join(pipeline_config_dict['arguments'])
```

Comment:
Why do we convert the arguments to a single string?

Reply:
I thought we are trying to create a single line command